modules/kubernetes-addons/keda/locals.tf

```diff
@@ -33,13 +33,15 @@ locals {
   ]
 
   irsa_config = {
-    kubernetes_namespace              = local.name
+    kubernetes_namespace              = local.helm_config["namespace"]
     kubernetes_service_account        = local.service_account_name
     create_kubernetes_namespace       = true
     create_kubernetes_service_account = true
     irsa_iam_policies                 = concat([aws_iam_policy.keda_irsa[0].arn], var.irsa_policies)
     irsa_iam_permissions_boundary     = var.irsa_permissions_boundary
     eks_cluster_id                    = var.addon_context.eks_cluster_id
+    iam_role_path                     = var.iam_role_path
```

Comment:
iam_role_path is not required for IRSA config

Reply:
Not required, but it is supported: https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/9684d0a106d43719e3f50adcdf3d6e0d9b4a85ca/modules/irsa/variables.tf#L41

I guess I'm wondering, how I would change it, as there doesn't appear to be an override/merge for `local.irsa_config` . In addition, if I change it for the `"aws_iam_policy" "keda_irsa"` would I not want it consistent across all the IRSA resources?